pkg/settings/blockchain_settings.go

```diff
@@ -104,6 +104,9 @@ type FunctionalitySettings struct {
 	MinUpdateAssetInfoInterval uint64 `json:"min_update_asset_info_interval"`
 
 	LightNodeBlockFieldsAbsenceInterval uint64 `json:"light_node_block_fields_absence_interval"`
+
+	GenerationPeriod uint64 `json:"generation_period"`
+	MaxEndorsements  int    `json:"max_endorsements"`
```

Comment:
Rename to `max_endorsers` according to the field in JSON settings.

Reply:
Why? It should be `max_endorsements` because number of endorsements is limited for a block, not number of endorsers for epoch as it was before. And by the way, we renamed `max_endorsers` to `max_endorsements` recently in configuration files.